### PR TITLE
Attestation refactor

### PR DESCRIPTION
**Pull request by anakin001**

### CODE REVIEW

File: src/modules/services/submodule/attestation/views/ViewServicesAttestationCreate.vue [hunk 1]
```diff
@@ -131,20 +186,89 @@ export default {
...
+      let fee = ""
+      if (isUploaded.value === false){
+        fee = "0.000000"
+      }else if (isUploaded.value === true && uploadType.value == "public"){
+        fee = "34.05000"
+      }else{
+        fee = "43.650000"
+      }
+      return fee
+    })
```
Comment (sleepyOwl14) [inline src/modules/services/submodule/attestation/views/ViewServicesAttestationCreate.vue:253]: can get the fee directly from created transaction ?

File: src/util/attestationUtils.ts [hunk 2]
```diff
@@ -0,0 +1,798 @@
...
+  return whiteListTransaction
+}
+
+export const hexStringToByte = (data: string) => {
+  if (!data) {
+    return new Uint8Array();
+  }
+  let a = [];
+  for (let i = 0, len = data.length; i < len; i += 2) {
+    a.push(parseInt(data.substring(i, i+1), 16));
```
Comment (sleepyOwl14) [inline src/util/attestationUtils.ts:222]: substring work differently from substr
it should be substring(i, i + 2)

File: src/models/REST/transaction.ts [hunk 3]
```diff
@@ -37,7 +37,7 @@ export class TransactionAPI {
      * @param transactionIds - Array of transactions id and/or hash.
      *
      */
-    getTransactions(transactionIds: string[]): Promise<Transaction[]>{
+    getTransactions(transactionIds: string[]): Promise<any>{
```
[the remark this answers is not in the document]
Comment (anakin001) [reply src/models/REST/transaction.ts:40]: the attestation code in angular tries to use Transaction.message.payload, which is a property only TransferTransaction type has. Also, i noticed angular changed it to any to bypass it.

File: src/util/attestationUtils.ts [hunk 4]
```diff
@@ -0,0 +1,798 @@
...
+        }
+        console.log('NOT_APOSTILLE')
+    }
+    public static hexStringToByte(str) {
+        if (!str) {
+            return new Uint8Array();
+        }
+        let a = [];
+        for (let i = 0, len = str.length; i < len; i += 2) {
+            a.push(parseInt(str.substring(i, i+1), 16));
```
[the remark this answers is not in the document]
Comment (anakin001) [reply src/util/attestationUtils.ts:56]: it was substr(i,2), which is from i to its next digit, substring takes length i + length i+1(which is its next digit). Correct me if im wrong

[the remark this answers is not in the document]
Comment (sleepyOwl14) [reply src/util/attestationUtils.ts:56]: you are wrong, the 2nd argument is the ending index, please test it.

File: src/models/REST/transaction.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (anakin001) [reply src/models/REST/transaction.ts]: fixed

**Review by sleepyOwl14 — APPROVED** [review]

**Pull request merged**